libs/sdk-ui-kit/src/Header/HeaderDataMenu.tsx

```diff
@@ -36,6 +36,7 @@ export const CoreHeaderDataMenu: React.FC<IHeaderDataMenuProps> = ({
             const { isDisable, tooltipText, isActive, className, key, onClick } = item;
             const classNames = cx(`gd-button-primary ${className}`, {
                 "is-active": isActive,
+                "is-normal": !isDisable && !isActive,
```

Comment:
should we check for isDisable in "is-active" class?

Reply:
If we just put the condition `!isActive` it could be isDisable state in below condition then the class name would be  "is-normal disabled"

Reply:
no, I'm talking about "is-active" 
should it be 
`"is-active": !isDisable && isActive`?

Reply:
I have fixed like suggestion thanks